templates/pages/events.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Events{% endblock %}
+
+{% block content%}
+
+
+	<div>
+		<h1><a href="">All CodeCatz Events</a></h1>
```

Comment:
:smile: here, have a :lollipop: :wink: 

just remove the a tags (and closing tags) and I'll merge it :+1: 
good work